src/main/java/com/netflix/nebula/hollow/ApiGeneratorTask.java

```diff
@@ -59,7 +59,7 @@ public void generateApi() throws IOException {
             mapper.initializeTypeState(clazz);
         }
 
-        String apiTargetPath = extension.destinationPath != null && extension.destinationPath.isEmpty() ? extension.destinationPath : buildPathToApiTargetFolder(extension.apiPackageName);
+        String apiTargetPath = !extension.destinationPath.isEmpty() ? extension.destinationPath : buildPathToApiTargetFolder(extension.apiPackageName);
```

Comment:
I stumbled upon this bug today as well.
Would it be easier to separate this bug fix out of the chicken and egg solution?